chaoslib/litmus/platform/gke/node_memory_consumption.yml

```diff
@@ -0,0 +1,149 @@
+ ## GETTING THE PRE CHAOS NODE NAME OF APPLICATION 
+- block:
+  
+    - name: Getting the pre-chaos pod name of application 
```

Comment:
Getting the application pod name ( pre chaos)

Reply:
Same thing